### DISCUSSION TITLE
Teyitli Bilgi Veren WhatsApp Gruplarından Bilgi Akışı

### DISCUSSION THREAD

**Discussion by ozcelikfu** [acikkaynak/deprem-yardim-projesi#62] [Ideas]
BEMTEG derneğinin bir whatsapp grubundayım 2700 küsür kişi ve 15 yöneticiden oluşuyor. Buradaki yöneticiler gelen duyuruları bizzat arayıp ulaşarak teyit ediyorlar. Teyitli bilgilerin otomatik çekilen bilgilerden daha değerli olduğunu düşündüğümden burada paylaşılan duyuruların bir şekilde buradaki projelere aktarılmasının faydalı olacağını düşünüyorum. Bunu yapmamız mümkün mü?

**Comment by sercangoger** [comment]
acil görülmesi gerekiyor bu durumun ozaman

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibinden şartlı kontrolünden geçti.
PM ekibine fikir iletildi.
Legal Ekip Notu: "legal altyapısı KVKK anlamında oluşturulabilir"

**Comment by ozcelikfu** [comment]
Bir gelişme olmadığını görüyorum. Ben https://twitter.com/TeyitliAfetDuy hesabından bu duyuruları şu an manuel olarak giriyorum. Twitter API erişimi açıldığında otomatiğe bağlayabilirim. Bu tweetlerdeki bilgileri dilerseniz projelerdeki veritabanlarına dahil edebilirsiniz
